merlin/config/celeryconfig.py

```diff
@@ -0,0 +1,29 @@
+"""
+Default celery configuration for merlin
+"""
+
+from merlin.log_formatter import FORMATS
+
+
+DICT = {
+    "task_serializer": "pickle",
```

Comment:
We can't change these currently or the system will break. Should we expose this?

Reply:
This isn't what's exposed; these are our defaults. However, with this PR technically everything is exposed, so users could break stuff in more ways than one.